x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -712,8 +712,8 @@ describe('when on the endpoint list page', () => {
       const renderResult = await renderAndWaitForData();
       const policyStatusLink = await renderResult.findByTestId('policyStatusValue');
       expect(policyStatusLink).not.toBeNull();
-      expect(policyStatusLink.getAttribute('href')).toEqual(
-        '/endpoints?page_index=0&page_size=10&selected_endpoint=1&show=policy_response'
+      expect(policyStatusLink.getAttribute('href')).toContain(
+        '/administration/endpoints?page_index=0&page_size=10&selected_endpoint=1&show=policy_response'
```

Comment:
I'm wondering why you need the `/administration` here and not in other tests if you are using `toContain` instead of `.toEqual`. Did I miss something?

Reply:
The `pathname` checked above is relative to the base path, the `href` here is absolute. But anyway you're right it is confusing. I changed everything to `isEqual` using path constants so it is explicit.

Reply:
Ahá! Got it, thanks for change this!